<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(222, 239, 231);</string>
  </property>
  <widget class="QPushButton" name="btninventory">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font Mono</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>LISTAR INVENTARIO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnpdf">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font Mono</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GENERAR PDF</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>101</y>
     <width>541</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font Mono</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="options">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font Mono</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>Seleccionar pelicula</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="store1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>96</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TIENDA UNO</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="store2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>96</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TIENDA DOS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textsearch">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>64</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font Mono</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>Buscar pelicula</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font Mono</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>REGRESAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnpdf_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font Mono</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>RESUMEN PDF</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
